Tasks/QuickPerfTest/Invoke-QuickPerfTest.ps1

```diff
@@ -184,22 +184,35 @@ function ComposeAccountUrl($vsoUrl)
     return $elsUrl
 }
 
+function ValidateInputs()
+{
+    if (![System.Uri]::IsWellFormedUriString($websiteUrl, [System.UriKind]::Absolute))
+    {
+        throw "Website Url is not well formed."
```

Comment:
Fair feedback. But as of now, none of the messages output by this task are localized. Also most of them are verbatim copies of the service side messages, which cannot be localized. Let me work with stakeholders here and get back to you on their recommendation.